### DISCUSSION TITLE
link between users under admin and users in teams

### DISCUSSION THREAD

**Discussion by giorgioUnimi** [pretix/pretix#4532] [General]
Hi to all, from documentation I don't understand the difference between users under admin and users under organizer->teams.
How to create a user that can work on a precise event for example as admin I create  an event and then that user go on to customize that event ?
thank you

**Comment by pc-coholic** [comment]
In the pretix backend, there is only users.

A user = one account with email/password, that is being used to access the pretix backend.

A user can be a member of none, one or more teams.

A team is an entity created on the event organizer level, that defines which users are allowed to perform certain actions (permissions) on all or only some events of an organizer.

By default, pretix will create an "Administrators"-team for every event organizer. Users that are a member of this team, will have full control of all events of that organizer.

In addition to that, user accounts can be flagged as "site admin". Giving a user this permission, they can enable "Admin mode" using a button in the top navigation bar. This will allow them to see and edit every event in the system - even if they would normally not have access to that specific event.

> How to create a user that can work on a precise event for example as admin I create an event and then that user go on to customize that event ?
 
The easiest way is to go to the teams page of the organizer account in question, and then invite the user by email to join a specific team (can be the admin-team, can be a new team with restricted permissions that you just created). If the user does not yet have a pretix account with that email-address, they will receive an email with a signup link; if they already have an account, they will be given the required permissions.
